libs/vscode/tasks/src/lib/nx-task-commands.ts

```diff
@@ -341,15 +354,58 @@ async function promptForList() {
 
 async function promptForMigrate() {
   const telemetry = getTelemetry();
-  telemetry.featureUsed('affected-cli');
-  const task = NxTask.create(
-    {
-      command: 'migrate',
-      flags: [],
-    },
-    cliTaskProvider.getWorkspacePath()
+  telemetry.featureUsed('migrate command');
+
+  const { workspacePath } = await getNxWorkspace();
+
+  const packageJson = await readAndParseJson(
+    join(workspacePath, 'package.json')
   );
-  tasks.executeTask(task);
+
+  if (
+    packageJson.dependencies == undefined ||
```

Comment:
currently this would exit even if `packageJson.dependencies` is defined because of the `||`. 
That's probably the wrong approach since only one of them can be defined and it's perfectly valid.

Reply:
You're right. I think I'll go with assigning an empty object in case one of the properties is `undefined`.